apps/passport-server/migrations/49_frogcrypto.sql

```diff
@@ -0,0 +1,19 @@
+create table frogcrypto_frogs (
+  id SERIAL PRIMARY KEY,
+  uuid UUID NOT NULL DEFAULT uuid_generate_v4(),
```

Comment:
i am assuming you need both an `id` and a `uuid` so that
1) the id is some coherent integer that the user can see
2) the uuid is an unguessable string used to access the image of a frog?

is that right?

also, idk if frogs should have uuids be default generated here, as opposed to being in the spreadsheet (i see uuids there)

Reply:
agree that uuid is probably better generated outside. updated the query

also on the fence whether we need both id and uuid but yes `id` is more user facing and `uuid` is for ~secret stuff like images. arguably they can both stuffed in the json but I'm leaning them are likely to be useful to queries and we want to keep them around forever.